src/api/public/apidocs-new/paths/source_project_name_package_name.yaml

```diff
@@ -1,3 +1,352 @@
+get:
+  summary: List source files of a package.
+  description: Get a list of source files belonging to a package.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: deleted
+      schema:
+        type: string
+      description: |
+        Set to `1` to list source files of deleted packages.
+
+        Return a "Bad Request" error (400) if it is set to `1` and the package exists.
+      example: 1
+    # TODO
+    # - in: query
+    #   name: emptylink
+    #   schema:
+    #     type: string
+    #     enum:
+    #       - 1
+    #       - 0
+    #   default: 0
+    #   description:
+    #   example: 1
+    - in: query
+      name: expand
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      default: 0
+      description: Expand links.
+      example: 1
+    - in: query
+      name: extension
+      schema:
+        type: string
+      description: List sources files with the given extension.
+      example: txt
+    - in: query
+      name: lastworking
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      default: 0
+      description: Show sources of last mergeable sources in case of conflicting changes.
+      example: 1
+    - in: query
+      name: linkrev
+      schema:
+        type: string
+      description: Show sources of the specified linked revision in the base package.
+    - in: query
+      name: meta
+      schema:
+        type: string
+        enum:
+          - 1
+          - 0
+      default: 0
+      description: Set to `1` to show only the metadata file (`_meta`) in the list of files.
+      example: 1
+    - in: query
+      name: product
+      schema:
+        type: string
+      description: Use with `view=products`. Limit the product view to a given product.
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Show sources of the specified revision.
+      example: 42
+    - in: query
+      name: view
+      schema:
+        type: string
+        enum:
+          - cpio
+          - getmultibuild
+          - info
+          - issues
+          - products
+          - productrepositories
+      description: |
+        Specify which information about a package should be returned.
+
+        * `cpio`: Stream all package files as cpio.
+        * `getmultibuild`: List multibuild packages.
+        * `info`: Show source version, md5sums and build description files, among other information.
+           See this [other endpoint](#/TODO) for details.
```

Comment:
This endpoint with `view=info` will be documented in other entry. Once we document it, we should link from here to the new documented endpoint.